src/components/Books.js

```diff
@@ -0,0 +1,22 @@
+import { AddBookForm } from './AddBookForm';
+
+export const Books = () => (
+  <div>
+    <ul>
+      <li>
+        <p>The Hunder Games</p>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <p>Dune</p>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <p>Capital in the Twenty-First Century</p>
+        <button type="button">Remove</button>
+      </li>
```

Comment:
- [ ] Our `Books` component should render a list of books using a `Book` component. You may consider initializing an array of books.
- [ ] Kindly use this [screenshot](https://user-images.githubusercontent.com/65476715/139659463-36c75e98-033d-4478-bb9f-42eaf6208f14.png) to set up the `Book` component.